expo-maps/ios/GoogleMapsView.swift

```diff
@@ -21,4 +21,20 @@ public final class GoogleMapsView: UIView {
   required init?(coder: NSCoder) {
     fatalError("init(coder:) has not been implemented")
   }
+  
+  
+  func mapType(mapType: MapType) -> Void {
```

Comment:
Actually, I'd rather go with the convention to prefix setters with `set`, so here it would be 👇 It's much more compliant with Swift/ObjC standards 😄 Probably the same is applicable to Kotlin, cc @lukmccall?
```suggestion
  func setMapType(_ mapType: MapType) -> Void {
```